qtrt/c.go

```diff
@@ -1,6 +1,6 @@
 package qtrt
 
 /*
-#cgo pkg-config: libffi
+// #cgo pkg-config: libffi
```

Comment:
don't comment it out - just add "linux" to indicate this is linux specific:
```suggestion
#cgo linux pkg-config: libffi
```